_episodes/10-singularity.md

```diff
@@ -99,7 +102,7 @@ export SINGULARITY_DOCKER_PASSWORD='mysecretpass'
 > > ## Solution
 > > ~~~bash
 > > export SINGULARITY_CACHEDIR="/tmp/$(whoami)/singularity"
-> > singularity shell -B /afs -B /eos -B /cvmfs docker://gitlab-registry.cern.ch/[repo owner's username]/[skimming repo name]:[branch name]-[shortened commit SHA] --docker-login
+> > singularity shell -B /afs -B /eos -B /cvmfs --docker-login docker://gitlab-registry.cern.ch/[repo owner's username]/[skimming repo name]:[branch name]-[shortened commit SHA]
```

Comment:
I had already wanted to open another issue for this. Non-CERN people or people running on their local machines will not have ``afs``, ``eos`` and ``cvmfs``. But it also doesn't seem to be necessary right?

Reply:
I propose to remove the ``.gitlab/build.sh`` thing (I assume that's why we needed the ``source /cvmfs/cms.cern.ch/cmsset_default.sh`` before?). And in the last example where we use our own docker image, the three mounts seem to be completely unnecessary anyway, right? (I can test this later)

Reply:
I am also not sure how useful the optional lesson will be for Non-CERN people as in my mind the point of the lesson is mainly as a work around to use containers on lxplus.

I can add that change, the .gitlab/build.sh doesn't really seem to matter.

Reply:
I think it's very relevant for non-CERN people as well! I think singularity is the dominant thing on most HPC facilities, right? I can run singularity on all servers that I have access to, but none of them will let me use docker. 

What exactly are the 3 mount points currently being used for?

Reply:
The cvmfs mount allows for access to the CERN virtual machine file service. Maybe I should use it in a separate example, it will be useful for people who want to load CMSSW for example. The eos mount allows for quick access to any eos files. The afs mount is just to get all of the normal stuff on lxplus.

But now that you point this out, I am thinking that maybe it would be better to separate these out a bit. I will add a new commit, I think I have a better idea of how to do this.